orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/recover_semaphore.sls

```diff
@@ -0,0 +1,12 @@
+{%- from 'postgresql/settings.sls' import postgresql with context %}
+
+{% set configure_remote_db = salt['pillar.get']('postgres:configure_remote_db', 'None') %}
+
+{% if 'None' != configure_remote_db %}
+
+recover-services-db-remote-required:
+  cmd.run:
```

Comment:
I think `file.exists` would be more suitable for this, and you won't need the `unless` part then

Reply:
`file.append` can be good, as I need to create the file or append it's content